<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>
 * Released as open source by Gabriel Caudrelier
 *
 * Developed by Gabriel Caudrelier, gabriel dot caudrelier at gmail dot com
 *
 * https://github.com/metrodango/pip3line
 *
 * Released under BSD 3-Clause see LICENSE for more information
</comment>
 <class>MemoryWidget</class>
 <widget class="QWidget" name="MemoryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="memInfo">
      <attribute name="title">
       <string>Memory Info</string>
      </attribute>
      <layout class="QVBoxLayout" name="memLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QCheckBox" name="refreshCheckBox">
            <property name="toolTip">
             <string>Activate regular memory refresh</string>
            </property>
            <property name="text">
             <string>Refresh memory</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="refreshRateSpinBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Refresh rate in ms</string>
            </property>
            <property name="minimum">
             <number>100</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="singleStep">
             <number>100</number>
            </property>
            <property name="value">
             <number>1000</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="procSelectWidget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="procLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Proc</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="procIDLabel">
            <property name="text">
             <string>Proc ID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="selectPushButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Select Process</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="mappingsTableWidget"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
